src/helpers/Url/Url.js

```diff
@@ -155,11 +155,21 @@ export const isCmsUi = memoize((currentPathname) => {
  */
 export function flattenHTMLToAppURL(html) {
   const { settings } = config;
+  const replacer = config.settings.prefixPath ?? '';
   return settings.internalApiPath
     ? html
-        .replace(new RegExp(settings.internalApiPath, 'g'), '')
-        .replace(new RegExp(settings.apiPath, 'g'), '')
-    : html.replace(new RegExp(settings.apiPath, 'g'), '');
+        .replace(new RegExp(settings.internalApiPath, 'g'), replacer)
+        .replace(new RegExp(settings.apiPath, 'g'), replacer)
+    : html.replace(new RegExp(settings.apiPath, 'g'), replacer);
+}
+
+export function addPrefixPath(src) {
```

Comment:
@sneridagh @nileshgulia1 I think this function is not needed. I put a `return src` on the first line of it, and everything worked fine.

Reply:
Although it works, without adding the prefix, the srcs and hrefs are missing the prefix in the generated HTML, which can be confusing. So it's better to keep the prefix.